### DISCUSSION TITLE
What Uniswap Universal Router commands are actually used?

### DISCUSSION THREAD

**Discussion by Elnaril** [Elnaril/uniswap-universal-router-decoder#33] [General]
I always wonder what should be the next functions this library should support, so I decided to list those that are actually used, and order them from the most to the least called.

[Full article and results here](https://www.reddit.com/user/E_l_n_a_r_i_l/comments/18mp50p/what_uniswap_universal_router_commands_are)

**Comment by Avkubs** [comment]
very interesting

**Comment by Elnaril** [comment]
The same analysis but 5 months later reveals some interesting results.
[Full article and results here](https://www.reddit.com/user/E_l_n_a_r_i_l/comments/1cwfsf8/what_uniswap_universal_router_commands_are/)
